whisper/cli/src/main.rs

```diff
@@ -215,8 +221,20 @@ fn execute<S, I>(command: I) -> Result<(), Error> where I: IntoIterator<Item=S>,
 	// Whisper protocol network handler
 	let whisper_network_handler = Arc::new(whisper::net::Network::new(pool_size, manager.clone()));
 
+	let network_config = {
+		let mut cfg = net::NetworkConfiguration::new();
+		let port = match &args.flag_port[..] {
+			"random" => 0 as u16,
+			port => port.parse::<u16>().expect("Invalid port specifier")
```

Comment:
I guess that is okay, but a dirty fix to prevent panic and handle the error could be:

```suggestion
						port => port.parse::<u16>().map_err(|e| e.to_string())?,

```